base/applications/regedit/find.c

```diff
@@ -93,9 +93,16 @@ CompareData(
     LPCWSTR psz1,
     LPCWSTR psz2)
 {
-    INT i, cch1 = wcslen(psz1), cch2 = wcslen(psz2);
+    ssize_t i;
```

Comment:
Using (unsigned) `size_t` here wouldn't work? Since you added the `cch1 >= cch2` test below, this should be OK I think.

Reply:
Fixed.